app/main/tv/codely/mooc/api/MoocApiApp.scala

```diff
@@ -47,10 +47,17 @@ object MoocApiApp {
 
     // let it run until user presses return
     println(s"Server online at http://$host:$port/\nPress RETURN to stop...")
+
     StdIn.readLine()
```

Comment:
We needed the console to stop in here, but it was never happening and the application was shutting down